rockcraft/lifecycle.py

```diff
@@ -49,7 +49,10 @@ def pack():
     # Obtain base image and extract it to use as our overlay base
     # TODO: check if image was already downloaded, etc.
     emit.progress(f"Retrieving base {project.base}")
-    base_image = oci.Image.from_docker_registry(project.base, image_dir=image_dir)
+    if project.base == "bare":
+        base_image = oci.Image.new_oci_image(image_dir=image_dir)
```

Comment:
to keep oci.Image, I would say project.base should also be passed in here, is tagging required in the called to logic?

Reply:
yes I think the tag plays a role, cause certain `umoci` calls do expect it. what do u propose?